app/src/main/java/com/project/symptoms/activity/MainActivity.java

```diff
@@ -283,29 +298,31 @@ private SymptomModel getNearestSymptomToCoordinates(float posX, float posY) thro
 
     }
 
-    private boolean isColorRed(int r, int g, int b){
-        return r == 255 && g == 0 && b == 0;
+    private boolean isColorRed(int r) {
+        boolean red = false;
+        if (r == 237 || r == 255 || r == 242 || r == 246) red = true;
```

Comment:
why only changed this for red?

Reply:
Because now are different types of red, so is the variable that is important to check and to don't do many "if" with the same logic.